### PR TITLE
Configure weeks to start on Sunday

### LINKED ISSUE #116: Configure weeks to start on Sunday
### What would you like to be added?

Make it possible for weeks to start on a Sunday in the Calendar view.

### Why is this needed?

Many countries start their week on a Sunday.

### PR DESCRIPTION
**Pull request by GamerGirlandCo**
closes #116

### CODE REVIEW

File: src/lib/stores/i18n.ts [hunk 1]
```diff
@@ -220,6 +220,11 @@ i18next.init({
               dayWithCount_one: "{{count}} day",
               dayWithCount_other: "{{count}} days",
             },
+            startOfWeek: {
+              name: "Start Week On",
```
Comment (marcusolsson) [inline src/lib/stores/i18n.ts:224]:
```suggestion
              name: "Start week on",
```

UI uses sentence casing.

File: src/views/Calendar/CalendarView.svelte [hunk 2]
```diff
@@ -67,6 +68,12 @@
 
   $: dateInterval = computeDateInterval(anchorDate, interval);
 
+  $: weekStart = config?.weekStart ?? "monday";
+  // $: dayjs.Ls[dayjs.locale()]!.weekStart = weekStart == "sunday" ? 0 : 1
```
Comment (marcusolsson) [inline src/views/Calendar/CalendarView.svelte:72]: Remove commented line.

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Thank you for this!

This seems to use Sunday as the default which would cause all existing users to switch, whether they wanted to or not. I'd prefer to keep the current behavior default to avoid frustration.

I tried it briefly and it seemed to have some trouble updating the locale initially. I had to switch a couple of times before it starting working. Will do some more testing.

Also, I think I need to reconsider putting the settings in the toolbar. It's starting to get a little cluttered with all the settings. Could you move the dropdown to the settings group on the right-hand side, to at least keep it consistent for now?

**Comment by GamerGirlandCo** [comment]
closing this and creating a new one since my new commits aren't showing.

**Pull request closed without merging**